code/src/cljs/sixsq/nuvla/ui/main/components.cljs

```diff
@@ -101,24 +101,28 @@
 (defn ErrorJobsMessage
   [_job-subs _set-active-tab-event _job-tab _on-click]
   (let [tr                (subscribe [::i18n-subs/tr])
-        errors-dissmissed (r/atom #{})]
+        error-dismissed   (r/atom false)]
```

Comment:
This will introduce a bug in following conditions:
Consider user has a job error being shown to him. He click dismiss on job error and do a new action on his deployment which fails. The `ErrorJobsMessage` component will not be reloaded and will keep error-dismissed value to true and then will not show the new error.

Reply:
Good catch! I _think_ I fixed it now. I'll test a bit tomorrow morning to make sure it works as intended. :)